mesheryctl/pkg/utils/formatter.go

```diff
@@ -46,63 +41,4 @@ func SetupMeshkitLogger(debugLevel bool, output io.Writer) {
 		os.Exit(1)
 	}
 	Log = logger
-}
-
-type Paginator struct {
-	data                 []string
-	lastVisibleItemIndex int
-	whiteBgPrinter       *color.Color
-	userMsg              string
-}
-
-func NewPaginator(header []string, rows [][]string) *Paginator {
-	var buf bytes.Buffer
-	renderedTableString := writeTableToBuffer(&buf, header, rows)
```

Comment:
Why these have been deleted?
If deleted, where they have been moved to?

Reply:
They were created for the pagination, it was redundant code. Also it was only showing one new model/component at a time. Now we are using the once method to print table.